en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,419 @@
+# Redis Connector Example
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker. It supports data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs and geospatial indexes with radius queries.
+
+This connector enables developers to use an external Redis server as a cache or a database in the WSO2 EI mediation logic.
```

Comment:
`cache or a database` shall we make this bold as it is the main use case?

Reply:
hyperloglogs --> hyperlogs?

Reply:
Updated.